data/saas/dataset/segment_dataset.yml

```diff
@@ -0,0 +1,133 @@
+dataset:
+  - fides_key: segment_connector_example
+    name: Segment Dataset
+    description: A sample dataset representing the Segment connector for Fidesops
+    collections:
+      - name: segment_user
+        fields:
+          - name: segment_id
+            data_categories: [user.derived.identifiable.unique_id]
+            fidesops_meta:
+              primary_key: True
+      - name: track_events
```

Comment:
Do you have any opinions around standardizing the collection names? My preference would be to use singular names for the collections.

Reply:
I think these were piped over from the historical segment yaml - I guess I don't understand why we need a standard internally here?  To me it's less confusing to match the third party source we are connecting to, then arbitrarily making them singular.

Reply:
Discussed, leaving as-is for now, we may have a broader discussion about this Friday.